pkg/apiserver/webhook/cloudentityselector_webhook.go

```diff
@@ -83,10 +83,10 @@ func (v *CESMutator) Handle(_ context.Context, req admission.Request) admission.
 	referencedAccount := &v1alpha1.CloudProviderAccount{}
 	err = v.Client.Get(context.TODO(), *accountNameSpacedName, referencedAccount)
```

Comment:
nit: Change to if err = (); err != nil {}

Reply:
Done